src/server/i18n/bg.json

```diff
@@ -0,0 +1,241 @@
+    "analytics_info": "Google Analytics stores information about:",
+    "analytics_text_list_one": "how you got to the site",
+    "analytics_text_list_two": "the pages you visit on Roadside Payments and government digital services, and how long you spend on each page",
+    "analytics_text_list_three": "what you click on while you're visiting the site",
+    "analytics_privacy": "Google doesn’t collect or store your personal information, such as your name or address, so this information can’t be used to identify who you are.",
+    "analytics_cookie_name_one": "_ga",
+    "analytics_cookie_purpose_one": "These help us count how many people visit Roadside Payments by tracking if you’ve visited before",
+    "analytics_cookie_expires_one": "2 years",
+    "analytics_cookie_name_two": "_gid",
+    "analytics_cookie_purpose_two": "These help us count how many people visit Roadside Payments by tracking if you’ve visited before",
+    "analytics_cookie_expires_two": "24 hours",
+    "analytics_cookie_name_three": "_gat",
+    "analytics_cookie_purpose_three": "Used to throttle request rate",
+    "analytics_cookie_expires_three": "1 minute",
+    "essential_cookies_title": "Essential cookies",
+    "essential_cookies_language_title": "Language translation",
+    "essential_cookies_language_text": "When you choose to use the Roadside Payments service using a language other than English, we'll set a cookie to remember your chosen language throughout your journey. This cookie does not store any personal data and is deleted when you close your browser.",
+    "essential_cookies_banner_title": "Cookies message",
+    "essential_cookies_banner_text": "You may see a banner when you visit Roadside Payments inviting you to accept cookies or review your settings. We’ll set cookies so that your computer knows you’ve seen it and not to show it again, and also to store your settings. We will store if you've chosen to accept or reject analytics cookies for 1 year.",
+    "language_cookie_name_one": "locale",
+    "language_cookie_purpose_one": "This lets us know which language you have selected so we can automatically display pages in your chosen language throughout your journey",
+    "language_cookie_expires_one": "When you close your browser",
+    "cookie_preferences_cookie_name_one": "cm-user-preferences",
+    "cookie_preferences_cookie_purpose_one": "Saves your cookie consent preferences",
+    "cookie_preferences_cookie_expires_one": "1 year",
+    "change_preferences_title": "Change your preferences",
+    "change_preferences_text": "You can <a href='/cookie-preferences'>change which cookies you’re happy for us to use.</a>",
+    "table_header_name": "Name",
+    "table_header_purpose": "Purpose",
+    "table_header_expires": "Expires"
+  }
+}
```

Comment:
No new line at end of file

Reply:
Please can you check the other files for this too

Reply:
Sure, will do.